operations/compare-helm-with-jsonnet/components/config/kustomization.yaml

```diff
@@ -68,3 +68,18 @@ patches:
     patch: |-
       - op: remove
         path: /config/memberlist
+
+  # Ignore tokens_file_path, except in ingester and store gateway
```

Comment:
This comment is a bit confusing. It took me some time to understand that it refers to the combination of the next two replace items. At the same time it doesn't really say much more than what's already below. Can we remove it?

```suggestion
```